README.md

```diff
@@ -1,12 +1,123 @@
 # SmartKickers
 
-Definition of Done
-
-- Working code
-- Created code need to be merged
-- Tests implemented
-- Tests passed
-- Code review by devs
-- Sanity check with mentors
-- Acceptance criteria fulfilled 
-- Work presented and accepted
+## Overview
+
+SmartKickers is a project to make your kickers more immersive. It uses Ximea camera to track the ball. 
+
+The following repository contains: 
+* Golang server for processing the data.
+* React client to display game information.
+
+You will also need SmartKickersAI provided by the developers per request. It contains:
+* Ball tracking software.
+* Node server responsible for sending data to the Golang server.
+
+Link to SmartKickerAI
+https://github.com/HackYourCareer/SmartKickersAI.
+
+## Features
+
+* Keeping track of the score.
+
+## Prerequirements
+
+* [python 3.10.5](https://www.python.org/downloads/macos/)
```

Comment:
That python dependency should be moved to the SmartKickersAI Readme.

Reply:
We will be adding additional Readme for SmartKickersAI. The same applies to the backend and frontend. This Readme aims to explain to the end-user in the simplest possible way how to run the whole application without having to open several Readme files.